samples/sampler/stories/default/RadioItem.js

```diff
@@ -13,17 +14,21 @@ RadioItem.displayName = 'RadioItem';
 storiesOf('Agate', module)
 	.add(
 		'RadioItem',
-		() => (
-			<div>
+		() => {
+			const slotBeforeSelection = select('slotBefore', ['', ...iconNames], Config);
+			const slotBefore = slotBeforeSelection ? (<Icon size="small" slot="slotBefore">{slotBeforeSelection}</Icon>) : null;
+
+			return (
 				<RadioItem
 					disabled={boolean('disabled', Config)}
 					icon={select('icon', ['', ...iconNames], Config)}
```

Comment:
`icon` prop has a default prop value in `RadioItemBase`, so we need to merge the metadata of `RadioItemBase`. In general, since we don't want to track down all props, we normally merge metadata from internally used components. What about updating Config like this?
> const Config = mergeComponentMetadata('RadioItem', ItemBase, Item, RadioItemBase, RadioItem);